sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -931,26 +913,18 @@ void setUncaughtExceptionHandler(UncaughtExceptionHandler handler) {
     }
 
     @VisibleForTesting
-    @Nullable
     ErrorReport buildErrorReport(ManagedErrorLog log) {
         UUID id = log.getId();
         if (mErrorReportCache.containsKey(id)) {
             ErrorReport report = mErrorReportCache.get(id).report;
             report.setDevice(log.getDevice());
             return report;
         } else {
-            File file = ErrorLogHelper.getStoredThrowableFile(id);
```

Comment:
getStoredThrowableFile is not used anymore, but I didn't see the removal of this function.